examples/gatsby/gatsby-config.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
     'gatsby-plugin-react-helmet',
     // If you want to use styled components you should add the plugin here.
     // 'gatsby-plugin-styled-components',
-    'gatsby-plugin-emotion',
```

Comment:
Does it mean we can remove `"gatsby-plugin-emotion"` from the package.json?

Reply:
Ooops, good catch! That's why we need your eyes 😄